common/src/main/java/com/microsoft/identity/common/internal/controllers/LocalMSALController.java

```diff
@@ -83,9 +83,11 @@
     private static final String TAG = LocalMSALController.class.getSimpleName();
 
     @SuppressWarnings(WarningType.rawtype_warning)
+    @EqualsAndHashCode.Exclude
```

Comment:
This may not be necceasary.

Reply:
Yeah as long as `onlyExplicitlyIncluded` is set to `true`